CHANGELOG.md

```diff
@@ -8,6 +8,7 @@
 ### Additions and Improvements
 - `--Xplugin-rocksdb-blockchain-blob-garbage-collection-enabled` Adds ability to enabled BlobDB GC for BLOCKCHAIN column family [#8599](https://github.com/hyperledger/besu/pull/8599)
 - `--Xplugin-rocksdb-blob-garbage-collection-age-cutoff`, `--Xplugin-rocksdb-blob-garbage-collection-force-threshold` BlobDB GC config options [#8599](https://github.com/hyperledger/besu/pull/8599)
+- Estimate gas on pending block by default [#8627](https://github.com/hyperledger/besu/pull/8627)
```

Comment:
is there a use-case for needing the previous behavior? ie is this a breaking change for some segment of users?

Reply:
I do not think so, because this can be seen as the correct behavior, so kind of a bug fix